megalinter/MegaLinter.py

```diff
@@ -822,7 +822,9 @@ def display_header():
         logging.info("")
 
     def check_results(self):
-        print(f"::set-output name=has_updated_sources::{str(self.has_updated_sources)}")
+        output_file = os.environ["GITHUB_OUTPUT"]
```

Comment:
check for existence of GITHUB_OUTPUT presence ^^ ( in case we are running in gitlab for example ^^ )